### PR TITLE
doc:Add possible values & '-f none' example

### PR DESCRIPTION
**Pull request by JangSoJin**
Hi, @namhyung :) 
I added more possible values(--color option, time_unit) and `uftrace replay -f none` example.
What do you think about these?

Signed-off-by: JangSoJin <wkdthwls3@naver.com>

### CODE REVIEW

File: doc/uftrace-live.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ The uftrace tool supports triggering actions on selected function calls with or
                      "recover" | "color="<color> | "time="<time_spec> | "read="<read_spec> |
                      "finish" | "filter" | "notrace"
     <time_spec>  :=  <num> [ <time_unit> ]
-    <time_unit>  :=  "ns" | "us" | "ms" | "s"
+    <time_unit>  :=  "ns"(= "nsec" ) | "us"(= "usec" ) | "ms"(= "msec" ) | "s"(= "sec") | "m"(= "min" )
```
Comment (namhyung) [inline doc/uftrace-live.md]: I prefer:

```
  <time_unit> := "ns" | "nsec" | "us" | "usec" | "ms" | "msec" | "s" | "sec" | "m" | "min"
```

**Comment by JangSoJin** [comment]
yes, Is it right? Thank you!!
* 30a1951  doc : Add `--color` possible values & `-f none` example
* 78b0152 doc : Add `time_unit` possible values 

Signed-off-by: JangSoJin wkdthwls3@naver.com

File: doc/uftrace-replay.md [hunk 2]
```diff
@@ -242,6 +242,19 @@ Each field has following meaning:
...
+     __cxa_atexit();
+     main() {
+       a() {
+         b() {
+           c() {
+             getpid();
+           } /* c */
+         } /* b */
+       } /* a */
+     } /* main */
```
Comment (namhyung) [inline doc/uftrace-replay.md]: I don't think this is needed.  It's not possible to have examples for every combination of options.  We already have an example for the -f option.

**Comment by JangSoJin** [comment]
I'm so sorry.. i made a mistake(because i cannot use `github` very well..) so i'll send you pull-request again.. :-( 
I'm so sorry.....

**Comment by namhyung** [comment]
No need to be sorry.  I appreciate your contribution.

**Pull request closed without merging**